rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/component/ComponentController.java

```diff
@@ -141,6 +143,7 @@ public ResponseEntity<CollectionModel<EntityModel<Component>>> getComponents(
             @RequestParam(value = "fields", required = false) List<String> fields,
             @Parameter(description = "Flag to get components with all details.")
             @RequestParam(value = "allDetails", required = false) boolean allDetails,
+            @RequestParam(value = "luceneSearch", required = false) boolean luceneSearch,
```

Comment:
Add @Parameter same like other params

Reply:
done

Reply:
@keerthi-bl could you take a look on my [comment](https://github.com/eclipse-sw360/sw360/pull/2201#issuecomment-1816034322)?

Reply:
@hoangnt2 , I have addressed that and changed the snippet accordingly, please have a look.